### DISCUSSION TITLE
How to setup probability forecast VAR in conf file

### DISCUSSION THREAD

**Discussion by BinbinZhou-NOAA** [dtcenter/METplus#1441] [Configuration] [answered]
I try to verify HREF's probability forecast. Take DPT as an example. Using wgrib2 to list HREF
probability output file, shown as following

35:8365670:d=2022021606:DPT:2 m above ground:6 hour fcst:prob >283.15
36:8658357:d=2022021606:DPT:2 m above ground:6 hour fcst:prob >285.93
37:8921389:d=2022021606:DPT:2 m above ground:6 hour fcst:prob >288.71
38:9162984:d=2022021606:DPT:2 m above ground:6 hour fcst:prob >291.48
39:9386748:d=2022021606:DPT:2 m above ground:6 hour fcst:prob >294.26 

Now I just verify  35:8365670:d=2022021606:DPT:2 m above ground:6 hour fcst:prob >283.15
From the MET document, the fcst entry could be setup like this
fcst = {
  field = [  { name = "PROB"; level = "Z2";
                  prob = { name = "DPT"; thresh_hi = 283.15; } } ] ; 
}

How is this setting to be translated to METplus conf file?
FCST_POINT_STAT_VAR1_NAME,  FCST_POINT_STAT_VAR1_LEVELS can specify name and level,
but don't know how to set up prob = { name = "DPT"; thresh_hi = "283.15; }} in the conf file?

**Comment by BinbinZhou-NOAA** [comment, marked as answer]
I found following setting is working

FCST_IS_PROB = true
FCST_PROB_IN_GRIB_PDS = true
BOTH_POINT_STAT_VAR1_NAME = DPT
BOTH_POINT_STAT_VAR1_LEVELS = Z2
BOTH_POINT_STAT_VAR1_THRESH = >283.15 

So this discussion can be closed

**Reply by j-opatz** [reply]
I'm glad to hear that you were able to solve this, Binbin. Also know that you can use the variables FCST_VAR1_LEVELS, FCST_VAR1_NAME, and FCST_VAR1_THRESH (along with the corresponding OBS_VAR1 settings) to make your METplus configuration file work.
